### PR TITLE
made the team name block

### PR DESCRIPTION
**Pull request by fakoredeDamilola**
Added a class of block to make the text a block text

### CODE REVIEW

**Comment by joepavitt** [comment]
This work is resolving https://github.com/flowforge/flowforge/issues/424

File: frontend/src/pages/team/dialogs/ConfirmTeamDeleteDialog.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
                                         Are you sure you want to delete this team? Once deleted, there is no going back.
                                     </p>
                                     <p class="text-sm text-gray-500">
-                                        Enter the team name '<code>{{team.name}}</code>' to continue.
+                                        Enter the team name '<code class="block">{{team.name}}</code>' to continue.
```
Comment (joepavitt) [inline frontend/src/pages/team/dialogs/ConfirmTeamDeleteDialog.vue:27]: I think we can also remove the `'` either side of the `<code>` block as it looks a little strange with them still in there:

Screenshot 2022-04-06 at 12 11 24

**Review by joepavitt — CHANGES_REQUESTED** [review]

File: frontend/src/pages/team/dialogs/ConfirmTeamDeleteDialog.vue (hunk 1 above)
Comment (fakoredeDamilola) [reply frontend/src/pages/team/dialogs/ConfirmTeamDeleteDialog.vue]: resolved

Comment (joepavitt) [reply frontend/src/pages/team/dialogs/ConfirmTeamDeleteDialog.vue]: Thanks! We had some other instances of this styling needing to be applied on a couple of other dialogs, mentioned back in the comment thread on #424 - if you can get those in for this PR then I think we'll be good to go after that

**Pull request merged**